platform/macos/src/Mapbox.h

```diff
@@ -1,10 +1,12 @@
 #import <Cocoa/Cocoa.h>
 
+#import "MGLFoundation.h"
+
 /// Project version number for Mapbox.
-FOUNDATION_EXPORT double MapboxVersionNumber;
+FOUNDATION_EXPORT MGL_EXPORT double MapboxVersionNumber;
```

Comment:
Make sure `MapboxVersionNumber` and `MapboxVersionString` are correctly set in the output build. I would hope that Xcode isn’t dependent on the particular syntax used to declare these constants, but you never know.